src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -552,6 +565,41 @@ class ProductaanvraagService @Inject constructor(
         assignZaak(createdZaak, zaakafhandelParameters)
         addInitiatorAndBetrokkenenToZaak(productaanvraag, createdZaak)
         cmmnService.startCase(createdZaak, zaaktype, zaakafhandelParameters, formulierData)
+
+        val to = extractEmail(extractIdentificatie(createdZaak))
+
+        val automaticEmailConfirmation = zaakafhandelParameters.automaticEmailConfirmation
+        if (automaticEmailConfirmation.enabled) {
+                val mailGegevens = MailGegevens(
+                    MailAdres(automaticEmailConfirmation.emailSender, null),
+                    MailAdres(to, null),
+                    MailAdres(automaticEmailConfirmation.emailReply, null),
+                    automaticEmailConfirmation.templateName.onderwerp,
+                    restMailGegevens.body,
+                    null,
+                    true
+                )
+                mailService.sendMail(mailGegevens, createdZaak.getBronnenFromZaak())
+                zaakService.setOntvangstbevestigingVerstuurdIfNotHeropend(createdZaak)
+        }
+    }
+
+    private fun extractIdentificatie(createdZaak: Zaak): String {
+        return when(val initiator = zaakService.getInitiatorFromZaak(createdZaak)) {
+            is MedewerkerIdentificatie -> { initiator.identificatie }
+            is NatuurlijkPersoonIdentificatie -> initiator.anpIdentificatie
```

Comment:
We do not support `anpIdentificatie` nor `annIdentificatie` fields in ZAC nor in the whole of PodiumD! Please do not use these fields!